utils/pre-swap-checker/package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "airswap-protocols-accessory",
+  "version": "0.0.1",
+  "repository": "git@github.com:airswap/airswap-protocols-accessory.git",
+  "license": "Apache-2.0",
+  "private": true,
+  "scripts": {
+    "clean": "rm -rf ./build",
+    "compile": "truffle compile",
+    "coverage": "node_modules/.bin/solidity-coverage",
+    "ganache": "ganache-cli -p 8545 --gasLimit 0xfffffffffff --time '2017-05-10T00:00:00+00:00' -s 0",
+    "hint": "yarn solhint \"./contracts/**/*.sol\"",
+    "lint": "yarn eslint \"./test/**/*.js\"",
+    "cp_migration_flat": "mkdir -p flatten/; cp contracts/Migrations.sol flatten/",
+    "test": "truffle test",
+    "flatten": "truffle run flatten",
+    "migrate": "yarn cp_migration_flat; truffle migrate --skip-dry-run",
+    "verify": "truffle run verify"
+  },
+  "dependencies": {
+    "@airswap/swap": "3.3.5",
+    "@airswap/tokens": "0.1.3",
+    "@airswap/types": "1.4.4",
+    "openzeppelin-solidity": "2.3"
+  },
+  "devDependencies": {
+    "@airswap/order-utils": "0.3.13",
+    "@airswap/test-utils": "0.1.3",
+    "@truffle/hdwallet-provider": "^1.0.17",
+    "babel-eslint": "^10.0.2",
```

Comment:
most of these are in the global package.json and so arent needed in here any more

Reply:
removing